remoteappmanager/db/interfaces.py

```diff
@@ -0,0 +1,81 @@
+from abc import ABCMeta, abstractmethod
+
+from traitlets import HasTraits, Unicode, Bool
+
+
+class ApplicationConfig(HasTraits):
```

Comment:
There is one advantage which is to be able to loop over the trait names when checking for 'equal'.  Without it we need to make sure we update the tests when we add attributes.  Alternatives?